testdata/ingress_alb/expected.cf

```diff
@@ -216,4 +216,4 @@
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
can you please cleanup these changes (do not commit these)?

Reply:
Isn't the newline better for `git diff`?

Reply:
@szuecs let's discuss this to see how to move forward.